internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -103,6 +105,7 @@ type Opts struct {
 	Confirm             bool
 	CurrentIP           bool
 	store               store.AtlasClusterQuickStarter
+	skipLogin           bool
```

Comment:
[optional] what do you think about naming this field `shouldRunLogin` here and and in setup (and having all assignments/checks reversed)? might add more clarity on what's its purpose

Reply:
done, thank you!